lib/test/t_RankSobolSensitivityAlgorithm_std.cxx

```diff
@@ -0,0 +1,67 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The test file of class SobolIndicesAlgorithm
+ *
+ *  Copyright 2005-2024 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/OT.hxx"
+#include "openturns/OTtestcode.hxx"
+
+using namespace OT;
+using namespace OT::Test;
+
+int main()
+{
+    RandomGenerator::SetSeed(0);
+
+    UnsignedInteger inputDimension = 3;
+    //UnsignedInteger outputDimension = 1;
+
+    RandomGenerator::SetSeed(0);
+    Description inputName(inputDimension);
+    inputName[0] = "X1";
+    inputName[1] = "X2";
+    inputName[2] = "X3";
+    Description formula(1);
+    formula[0] = "sin(pi_*X1)+7*sin(pi_*X2)*sin(pi_*X2)+0.1*((pi_*X3)*(pi_*X3)*(pi_*X3)*(pi_*X3))*sin(pi_*X1)";
+
+    SymbolicFunction model(inputName, formula);
+
+    JointDistribution::DistributionCollection marginals(inputDimension);
+    marginals[0] = Uniform(-1.0, 1.0);
+    //     marginals[0].setDescription("Marginal 1");
+    marginals[1] = Uniform(-1.0, 1.0);
+    //     marginals[1].setDescription("Marginal 2");
+    marginals[2] = Uniform(-1.0, 1.0);
+    //     marginals[2].setDescription("Marginal 3");
```

Comment:
use the ctor that initializes with list size and element:
marginals(inputDimension, Uniform(-1.0, 1.0))